versions.yml

```diff
@@ -16,3 +16,7 @@ windows_utilities_version: '0.11.0'
 windows_utilities_sha1: 'efc10ac0f4acae23637ce2c6f864d20df2e3a781'
 bbr_sdk_version: '1.15.0'
 bbr_sdk_sha1: 'b2d8584dd2ed964c4849cb6d7b536e6cea3e6e8d'
+uaa_version: '74.9.0'
```

Comment:
Is there any specific reason why we can't go with the latest release here? (v74.14.0, according to https://bosh.io/releases/github.com/cloudfoundry/uaa-release?all=1)

Just wondering it it's a compatibility issue.

Thanks!

Reply:
More it is the one we tested against, because it was the one documented in your docs at the moment.

https://docs.pivotal.io/p-concourse/v5/advanced/integrate-credhub-uaa-bosh/#install-credhub-uaa-with-bosh
Section #3